semver/is_semver_test.ts

```diff
@@ -47,6 +45,8 @@ Deno.test({
   fn: async (t) => {
     let i = 0;
     const versions: [unknown][] = [
+      [{ major: 0, minor: 0, patch: 0, prerelease: [] }],
+      [{ major: 0, minor: 0, patch: 0, build: [] }],
```

Comment:
We also need to add test cases to check with both prerelease and build undefined

Reply:
done